packages/web3torrent/src/clients/web3torrent-client.ts

```diff
@@ -51,13 +51,14 @@ export const upload: (input: WebTorrentSeedInput) => Promise<ExtendedTorrent> =
   );
 };
 
-export const cancel = (id: string = '') => {
+export const cancel = (torrentId: string = '') => {
```

Comment:
Note that when the `DownloadInfo` react component calls this function, there is wait or callback for the action to complete. Is that the intention?

Reply:
this function call is useless, I've got a bunch of things that I want to refactor about this function.
The `web3torrent.cancel()` function is `async` (both in this PR and in master), so this function having a callback is only used to show errors (and using a callback for that is useless in this case).
I think that initially was the function was set up this way because the `webtorrent.cancel` has a callback, but right now it doesn't make any sense as it is.

My plan is to refactor this `cancel()` function, the new `pause()` and the `resumeTorrent()` ones, on another PR.
I want to merge this PR, it's getting kind of old and large, and I want to make sure that cancelling works consistently, at least.